src/services/documents_utils.py

```diff
@@ -0,0 +1,30 @@
+# defining functions that will manage communication with s3 bucket
+import boto3
+
+
+def upload_file_to_s3(bucket_name: str, key: str, bin_file: bytes):
+    s3 = boto3.resource('s3')
+    bucket = s3.Bucket(bucket_name)
+    try:
+        bucket.put_object(Key=key, Body=bin_file)
+    except Exception as ex:
+        print(ex)
+        raise ex
+    else:
+        return True
+    
+
+def download_file_from_s3(bucket_name: str, key: str):
+    s3 = boto3.resource('s3')
+    try:
+        return s3.Object(bucket_name=bucket_name, key=key).get()['Body'].read()
+    except Exception as ex:
+        raise ex
+    
+
+def delete_file_from_s3(bucket_name: str, key: str):
+    s3 = boto3.resource('s3')
+    try:
+        return s3.Object(bucket_name=bucket_name, key=key).delete()
+    except Exception as ex:
+        raise ex

```

Comment:
Additional comment for the whole project - don't forget about logging. The first thing you will do if you come across the error on your environment - read the logs
You create a new record in DB - put logger.info("Creating document with id={id}")
You download smth from S3 - put logger.info("Downloading the file with key={key}")
Logging will save you time in the future for fast understanding in which piece of code the error happens